API/common/utils.py

```diff
@@ -58,7 +59,8 @@ def execute(cwd, cmd, args=None, quiet=False):
         return output, exitcode
 
     except Exception as e:
-        console.fail('[Failed - %s %s] %s' % (cmd, ' '.join(args), str(e)))
+        console.log('[Failed - %s %s] %s' % (cmd, ' '.join(args), str(e)), '\033[1;31m')
+        sys.exit(exitcode)
```

Comment:
console.fail uses `exit(1)` that's why the return value should be 1 if error happens. Do you need the return value of the concrete subprocess?